topic_resources/see-more.md

```diff
@@ -29,58 +29,56 @@ A **feed** is an API that a web service uses to provide users with frequently up
 Build a feed aggregator service that allows users to login via Twitter, Github, Instagram and Vimeo. It will allow users to view updates from all their tracked feeds in one location, and share them back to associated social media services.
 
 ### Technical Requirements
-#### Bronze (20 points)
+#### Wave 1
 + Allows Users to sign in and out using [OmniAuth's Developer Strategy](http://rubydoc.info/github/intridea/omniauth/master/OmniAuth/Strategies/Developer)
 - Can search Twitter users
 - Can add Twitter users to your SeeMore feed
 - Can search for a Vimeo user
 - Can add Vimeo users to your SeeMore feed
-+ Can aggregate feeds from two separate social media feeds (Twitter & Vimeo)
+- Can aggregate feeds from two separate social media feeds (Twitter & Vimeo)
 - Write at least 20 specs (5 from each member)
 
-#### Silver (20 points)
-All of Bronze, plus:
+#### Wave 2
 - Save each social media post to the local database, duplicates should not be allowed.
-+ Allows Users to sign-up and login with at least one social media account (Instagram (required), Twitter, Vimeo, Github)
+- Allows Users to sign-up and login with at least one social media account (Instagram (required), Twitter, Vimeo, Github)
 - Can search and add Github users to your SeeMore feed
 - Can search and add Instagram users to your SeeMore feed
-+ Can aggregate feeds from
+- Can aggregate feeds from
     - One or more Twitter users
     - One or more Vimeo users
     - One or more Github users
     - One or more Instagram users (must be authenticated with Instagram)
 
-#### Gold (20 points)
-All of Silver, plus:
+#### Wave 3
 - Automatically pulls in Twitter timeline feed for the authenticated user
 - Automatically pulls in Instagram timeline feed for the authenticated user
-+ Uses **cron** to periodically update feeds without duplication
-+ Can send email or SMS messages to users to update them
+- Uses **cron** to periodically update feeds without duplication
+- Can send email or SMS messages to users to update them
```

Comment:
Not sure this is worthwhile; we haven't talked about sending emails at all. I guess folks could look into ActionMailer, but it's kind of a pain to set up. Alternately, it could be another API service for them to consume, like Twilio.